packages/mobile/src/geth/RNGethSigner.ts

```diff
@@ -0,0 +1,91 @@
+import { normalizeAddressWith0x } from '@celo/utils/src/address'
+import { Tx } from 'web3-core'
+import { Signer } from '@celo/contractkit/lib/wallets/signers/signer'
+import {
+  encodeTransaction,
+  extractSignature,
+  RLPEncodedTx,
+  rlpEncodedTx,
+} from '@celo/contractkit/lib/utils/signing-utils'
+import RNGeth from 'react-native-geth'
+import * as ethUtil from 'ethereumjs-util'
+
+const INCORRECT_PASSWORD_ERROR = 'could not decrypt key with given password'
+const currentTimeInSeconds = () => Math.round(Date.now() / 1000)
+
+/**
+ * Implements the signer interface on top of a RNGeth instance
+ */
+export class RNGethSigner implements Signer {
+  /**
+   * Construct a new instance of the RPC signer
+   *
+   * @param geth the RNGeth instance
```

Comment:
It seems like this RNGethSigner still requires the geth light client to be initialized. Is the plan to implement another signer that does not use geth?

Reply:
Resolving based on our chat